<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT Open CV &amp; CL TEST</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cb_ExampleList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>401</width>
      <height>27</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="minimumContentsLength">
     <number>0</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>OCL LIBRARY PERFORMANCE</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bt_Start">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_OpType">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>CL On&amp;&amp;Off</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="rb_cl_on">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CL ON</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_cl_off">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CL OFF</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_Result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>661</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>RESULT</string>
    </property>
    <widget class="QTextEdit" name="te_Result">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>641</width>
       <height>401</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_Version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;OpenCV Version : &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_Stop">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_oclList">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>ocl Library List</string>
    </property>
    <widget class="QComboBox" name="cb_LibraryList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>BlendLinear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sobel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GaussianBlur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multiply</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CvtColor</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
